js/count-letter-in-string.js

```diff
@@ -0,0 +1,17 @@
+// https://www.codewars.com/kata/52efefcbcdf57161d4000091/train/javascript
+
+function count(string) {
+  const lettersArr = string.split('');
+  let lettersCount = {};
+
+   lettersArr.forEach((i) => {
+    if (lettersCount[lettersArr[i]] === undefined) {
```

Comment:
No, now it won't work. In this case, `i` is not the index. `i` is the value

Please analyze it thoroughly :slightly_smiling_face: